tests/test_gestalt.py

```diff
@@ -7,6 +7,49 @@
 import hvac
 
 
+# Testing member function
+def test_combined_into():
+    g = gestalt.Gestalt()
+    dictionary = {}
+    dictionary_ = {} 
+    dict1 = {
+        "local": 1234,
+        "pg": {
+            "host": "dict1_pg",
+            "pass": "dict1_pg"
+        }
+    }
+    dict2 = {
+        "local": 1234,
+        "pg": {
+            "host": "dict2_pg"
+        }
+    }
+
+    g.combine_into(dict1, dictionary)
```

Comment:
I think it may be god to add a test for `g.combine_into({}, some_dict)` to test that adding an empty dict does not corrupt the combined dict. Also a test to check that keys at the top level are properly overridden would be good

Reply:
Gestalt tests also test the top-level ones and the public tests check many things scenarios.